apps/events/views.py

```diff
@@ -1,10 +1,29 @@
 from django.http import HttpResponse
-from django.shortcuts import render
+from django.shortcuts import render, redirect
+from django.contrib.auth.decorators import permission_required
+from .models import Event
+from django.contrib import messages
```

Comment:
Arrange the imports in ascending order. Like, django.contrib... should come before django.http